### DISCUSSION TITLE
Integration with PlantUML diagrams

### DISCUSSION THREAD

**Discussion by Camille019** [typst/typst#1851] [Ideas]
Add the ability to write [PlantUML](https://plantuml.com) code in Typst document and call PlantUML to generate/insert the corresponding images into the final document.

This can be another good candidate for a plugin once Typst supports it.

**Comment by ensko** [comment]
> once Typst supports it

That seems to be [now :tada:](https://github.com/typst/typst/releases/tag/v0.8.0)! I'm too busy to work on such a plugin myself right now, but putting it out there if someone is inspired!

**Comment by Marcono1234** [comment]
Related to #161 (see also #1421 and #3090 for potential issues specific to Mermaid).

There is https://typst.app/universe/package/pintorita/ which uses [Pintora](https://github.com/hikerpig/pintora), and that describes itself as "Heavily inspired by Mermaid.js and PlantUML". But unfortunately that Typst package currently has not very good performance, see https://github.com/taylorh140/typst-pintora/issues/2.
